sys/include/ztimer/xtimer_compat.h

```diff
@@ -105,12 +112,28 @@ static inline xtimer_ticks64_t xtimer_ticks_from_usec64(uint64_t usec)
 
 static inline xtimer_ticks32_t xtimer_now(void)
 {
-    return ztimer_now(ZTIMER_USEC);
+    if (IS_USED(MODULE_ZTIMER_USEC)) {
+        return ztimer_now(ZTIMER_USEC);
+    }
+    else if (IS_USED(MODULE_ZTIMER_MSEC)) {
+        return ztimer_now(ZTIMER_MSEC) * US_PER_MS;
+    }
+    else {
+        return 0;
```

Comment:
```suggestion
        extern void xtimer_now_called_but_no_backend_available(void);
        xtimer_now_called_but_no_backend_availa();
```

Maybe better have a link time error than strange run time behaviour?

Reply:
Is this not what the `_XTIMER_BACKEND_NOT_IMPLEMENTED` is? Should it be called here?